contrib/examples/actions/mistral-handle-retry.yaml

```diff
@@ -0,0 +1,30 @@
+version: '2.0'
+name: "examples.mistral-handle-retry"
+
+workflows:
+
+    main:
+        type: direct
+        tasks:
+            task1:
+                workflow: work
+                policies:
+                    retry:
+                        count: 20
+                        delay: 3
+
+    work:
+        type: direct
+        tasks:
+            do:
+                action: core.local
+                input:
+                    cmd: "if [ ! -e \"/tmp/done\" ]; then exit 1; fi"
+                on-error:
+                    - undo
+            undo:
+                action: core.local
+                input:
+                    cmd: "echo \"File does not exist.\""
+                on-success:
```

Comment:
yeah, the thing is what if the undo failed...  right now, the whole thing retries regardless.   so the undo WF must be pretty reliable.